src/coreclr/src/libraries-native/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CORECLR_BUILD ON)
+
+if(CLR_CMAKE_HOST_UNIX AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD)
+    add_subdirectory(../../../libraries/Native/Unix/System.Globalization.Native System.Globalization.Native)
```

Comment:
Is the binary_dir argument necessary here? We don't use such a convention anywhere

Reply:
If we don't specify it, then it would be under `libraries-native` in the intermediate directory instead?

Reply:
We need to specify a directory because the subdirectory that we're adding is not a subdirectory of src/coreclr/src/libraries-native.

Reply:
I left it as is as without it it breaks, any suggestion, I would appreciate it.

Reply:
I see, cmake complains about it. The CMake doc was a bit unclear on this, it seemed to me as if the non-relative part of the src path would be used relative to the current bin dir, but that's obviously not the case.